Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -892,25 +893,31 @@ namespace AzToolsFramework
     {
         if (!m_prefabsAreEnabled)
         {
-            return m_isLevelEntityEditor ? InspectorLayout::LEVEL : InspectorLayout::ENTITY;
+            return m_isLevelEntityEditor ? InspectorLayout::OpenPrefabContainerEntity : InspectorLayout::Entity;
```

Comment:
Same here. The naming needs to be improved here I guess. When prefabs are disabled, why are we relying on an enum called OpenPrefabContainerEntity?

Reply:
It doesn't display the transform component, and it only lets you add level components :D

Reply:
Done :)